cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/MonitorUnitTestUtils.java

```diff
@@ -44,6 +46,13 @@ private MonitorUnitTestUtils() {
     return NODES.clone();
   }
 
+  /**
+   * @return The clone of the nodes in the {@link #getCluster(Collection)} and {@link #getMetadata(Collection)}.
+   */
+  public static List<Integer> nodeIds() {
+    return Arrays.asList(NODES.clone()).stream().map(n -> n.id()).collect(Collectors.toList());
```

Comment:
how about 
```suggestion
    return Arrays.stream(NODES).map(Node::id).collect(Collectors.toList());
```
imho array cloning is unnecessary also more concise